docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
+variables set. Can be used to create combined commands.
+
```

Comment:
This should include the command format, e.g., `tsh exec [--shell] -- <command>`. Other command entries in this reference add this after the first paragraph.